namespaces/live-1.cloud-platform.service.justice.gov.uk/cccd-dev/resources/variables.tf

```diff
@@ -0,0 +1,35 @@
+variable "domain" {
+  default = "cccd.dev.laa-get-paid.service.gov.uk"
```

Comment:
can you please look in to the guidance and change the domain name accordingly
```https://user-guide.cloud-platform.service.justice.gov.uk/tasks.html#using-a-custom-domain```

Reply:
Thanks @vijay-veeranki-moj. 

From reading the guidance I seem to need to, and could you confirm:

1. add a certificate resource, etc...
2. ideally use a subdomain of the DNS ZONE/HOSTED ZONE - `claim-crown-court-defence.service.gov.uk`

On point 2, because we are migrating an existing template-deploy app:
2.1. we already have a DNS ZONE of `claim-crown-court-defence.service.gov.uk` for the live/prod service (on TD) so, I assume, this would mean we cannot create another such zone and therefore no subdomains of it e.g. `cccd.dev.claim-crown-court-defence.service.gov.uk`. Is that right?

2.2. Our existing TD pre-prod environments use a DNS ZONE of `dsd.io` with subdomains for the various environments (e.g. `staging-adp.dsd.io`). **Does it, therefore, sound sensible to create a separate pre-prod DNS ZONE** of say `claim-crown-court-defence.service.justice.gov.uk` - i.e. adding the `justice` to differentiate. other suggestions welcome! and then subdomains following the [naming Pre-production environments domains standard](https://ministryofjustice.github.io/technical-guidance/standards/naming-domains/#pre-production-environments)
e.g.`cccd.dev.claim-crown-court-defence.service.justice.gov.uk`

2.3 Possibly thinking to far ahead but as an idea, does it sound sensible/possible to create a `claim-crown-court-defence.service.**justice**.gov.uk` DNS ZONE (as mentioned above) and use it to set up the intended prod environment too. Once migration (RDS/S3) is complete would it then be possible to remove the old TD DNS ZONE and amend the `claim-crown-court-defence.service.**justice**.gov.uk` to `claim-crown-court-defence.service.gov.uk

if easier, I can organise a meeting with whomever

Reply:
Ya good idea, let us have a meeting

Reply:
thanks @vijay-veeranki-moj, as discussed, let me know when you guys can schedule a meeting.

Reply:
Hi, @vijay-veeranki-moj have amended this PR if you could take a look. I will PR the production HOSTED ZONE changes we talked about separately